doc/source/user_guide/documentation/icepyx.rst

```diff
@@ -23,4 +23,5 @@ Diagrams are updated automatically after a pull request (PR) is approved and bef
    query
    read
    quest
+   auth
```

Comment:
Wondering at the "level" auth should be given within the docs. It seems like it might be more appropriate to put it into the components list (via `components.rst`). Thoughts?